airflow-core/src/airflow/migrations/versions/0070_3_1_0_change_deadline_to_utc.py

```diff
@@ -0,0 +1,56 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Change Deadline in the Dag table from DateTime to UTC DateTime.
```

Comment:
It looks like you're modifying the deadline table below, not the dag table.

Reply:
It's not like I wrote my intention RIGHT IN THE DOCSTRING.  🤦‍♂️